README.md

```diff
@@ -46,12 +104,14 @@ The meaning of these values are:
 
 If `options.getRatings` is not true, then this value will always be `0`.
 
+* * *
+
 ## Contributing
 
 All contributions are accepted! If you think you can bring uploading support, or make the lib perform better, make a PR and start coding!
```

Comment:
You supplied a contribution guideline yet it wasn't hotlinked in the README

Reply:
Linked it in the README as of 744f702235e2327ca141f8f4aab1ad45c5d5ab71